### DISCUSSION TITLE
Research Branch

### DISCUSSION THREAD

**Discussion by charlieikosi** [pinpointltdpng/ODFD-PROJECT#4] [Announcements]
We have a branch solely for storing notes and research material called "research_notes". I have already uploaded several documents on there for you to access. There are no permissions on this branch so you can push your commits to it.

I highly encourage you all to utilize this branch to share knowledge and resources.

**Comment by anapitalai** [comment]


**Reply by anapitalai** [reply]
One interesting feature of github is ACTIONS, for CI/CD, I am currently using them in my web designed pipelines, I currently learnt about this feature and love what it offers.

**Comment by anapitalai** [comment]
I have cloned the repo to my local machine, these are the steps, to be able to do the steps, you need git installed locally and it configured with your credentials for github.
git clone <repo_name>               #this clones the main n resource_notes branches
cd  git_repo_name
#below are  commands to work with git
git log
git branch -a               #view all branches
git checkout resource_notes        #change branch heads
#add contents to resource_notes
git merge     #to merge changes in resource_notes to main, change to main and merge
git push origin main
git clone --branch <branch_name> --single-branch  <repo_name>   #clone a single branch 
### More is involved

**Reply by charlieikosi** [reply]
Thanks @anapitalai for this. Create a folder in the research_notes branch and load these notes there
